### ISSUE TITLE
Fix external link icon contrast ratio in Smartmet Alert client

### ISSUE THREAD

**Issue by geosaaga** [fmidev/weather-app#1201]
According to AVI accessibility monitoring report there is insufficient contrast ratio in external link icons in Smartmet Alert client. Update client to application when fix has been made.
